web/src/main/java/it/pagopa/selfcare/mscore/web/controller/InstitutionController.java

```diff
@@ -124,6 +124,28 @@ public ResponseEntity<InstitutionResponse> createInstitutionFromIpa( @RequestBod
         return ResponseEntity.status(HttpStatus.CREATED).body(institutionResourceMapper.toInstitutionResponse(saved));
     }
 
+    /**
+     * The function create an institution retriving values from ANAC
+     *
+     * @param institution InstitutionRequest
+     * @return InstitutionResponse
+     * * Code: 201, Message: successful operation, DataType: InstitutionResponse
+     * * Code: 404, Message: Institution data not found on Ipa, DataType: Problem
+     * * Code: 400, Message: Bad Request, DataType: Problem
+     * * Code: 409, Message: Institution conflict, DataType: Problem
+     */
+    @ResponseStatus(HttpStatus.CREATED)
+    @ApiOperation(value = "${swagger.mscore.institution.create.from-anac}", notes = "${swagger.mscore.institution.create.from-anac}")
+    @PostMapping(value = "/from-anac/")
+    public ResponseEntity<InstitutionResponse> createInstitutionFromAnac(@RequestBody @Valid InstitutionRequest institution) {
```

Comment:
Why we need an InstitutionRequest if we use data from SaResource for saving institution on createInstitutionFromAnac?

Reply:
i have looked at CreateInstitutionStrategyAnac and institution fields aren't used....try to check there